content/balance/2020-12-08-9.0_FAQ.md

```diff
@@ -519,8 +523,18 @@ Currently with the new legendary {{< item 355315 "Sinful Hysteria" >}} it outsim
 
 Not currently competitive.
 
+  
+  
+<div id="domination-sockets">
+
+## [What Domination Sockets should I use?](#domination-sockets)
 
+</div>
 
+With the current iteration of Domination Sockets you should use the Chaos Bane set. After you get the set, you should look to get the Bek shard, which is a 1% dps increase and as your last shard use the Cor shard, which is the lowest dps increase with 0.5%.
```

Comment:
Should be noted that Bek has far higher value when the important damage phases are below ~40% hp and Shard of Cor is even better with common add spawns.